addons/stock/models/stock_rule.py

```diff
@@ -571,7 +570,7 @@ def _get_rule(self, product_id, location_id, values):
         # Get a mapping (location_id, route_id) -> warehouse_id -> rule_id
         rule_dict = self._search_rule_for_warehouses(
             values.get("route_ids", False),
-            values.get("product_packaging_id", False),
+            values.get("uom_id", False),
```

Comment:
I think there's something fishy going on between `uom_id` and `packaging_uom_id` here. :eyes: